packages/core/local-require/src/localRequire.js

```diff
@@ -1,29 +1,39 @@
 // @flow strict-local
 
 import type {FilePath, PackageJSON} from '@parcel/types';
+import type {WorkerApi} from '@parcel/workers';
 
-import installPackage from '@parcel/install-package';
+import installPackage, {
+  installPackageFromWorker
+} from '@parcel/install-package';
 import {dirname} from 'path';
 
 import {resolve} from '@parcel/utils';
 
 const cache: Map<string, [string, ?PackageJSON]> = new Map();
 
-export default async function localRequire(
+export async function localRequireFromWorker(
```

Comment:
heh, this is interesting. I guess it's fine for our internal use, but what about plugins? Should they know if they are running in a worker or not?

Reply:
They shouldn't, no. That's why we pass `localRequire` to certain apis (`getConfig` and `transform` in transformers for now) for plugins. I don't think they should be requiring it directly.

Another option would be to expose `childState` or something like it from the worker package, but that wouldn't be compatible with different versions of workerfarm being present.